pulp_container/app/tasks/synchronize.py

```diff
@@ -77,3 +82,23 @@ def pipeline_stages(self, new_version):
         ]
 
         return pipeline
+
+    def create(self):
+        """
+        Perform the work. This is the long-blocking call where all syncing occurs.
+
+        Returns: The created RepositoryVersion or None if it represents no change from the latest.
+        """
+
+        # This method is copied from pulpcore with the addition of maxsize=100 to circumvent sync
+        # issues with remote artifact creation.
+        with tempfile.TemporaryDirectory(dir="."):
+            with self.repository.new_version() as new_version:
+                loop = asyncio.get_event_loop()
+                stages = self.pipeline_stages(new_version)
+                stages.append(ContentAssociation(new_version, self.mirror))
+                stages.append(EndStage())
+                pipeline = create_pipeline(stages, maxsize=100)
```

Comment:
if you're talking specifically about reverting the change then it should be 1000